### DISCUSSION TITLE
S3&Knuckles Seamless Act transitions

### DISCUSSION THREAD

**Discussion by FriskTheFallenHuman** [Techokami/SonicWorldsNext#55] [Q&A] [answered]
I Wonder if the project would have a Sonic 3 & Knuckles Seamless Act transitions?

**Comment by DimensionWarped** [comment, marked as answer]
We've discussed a bit of this internally and have some ideas on how to implement it, but so far they aren't available just yet. Long story short is that it's doable, but there are probably some rough edges that need to be worked out.

**Comment by DimensionWarped** [comment]
btw, it's remarkable how these two things are actually the same problem when you think about it.

-----> https://github.com/Techokami/SonicWorldsNext/discussions/42
